web/client/components/I18N/HTML.jsx

```diff
@@ -11,23 +11,14 @@ var FormattedHTMLMessage = ReactIntl.FormattedHTMLMessage;
 
 var Message = React.createClass({
     propTypes: {
-        locale: React.PropTypes.string,
-        messages: React.PropTypes.object,
         msgId: React.PropTypes.string.isRequired,
         msgParams: React.PropTypes.object
     },
     contextTypes: {
-        locale: React.PropTypes.string,
-        messages: React.PropTypes.object
+      intl: React.PropTypes.object
     },
     render() {
-        var locale = this.props.locale || this.context.locale;
-        var messages = this.props.messages || this.context.messages;
-        let message = messages;
-        this.props.msgId.split('.').forEach(part => {
-            message = message[part];
-        });
-        return <FormattedHTMLMessage locales={locale} message={message} {...this.props.msgParams}/>;
+        return this.context.intl ? <FormattedHTMLMessage id={this.props.msgId} values={this.props.msgParams}/> : <span>{this.props.msgId || ""}</span>;
```

Comment:
id should not be used here, we can have duplicated htmlMessages with the same msgId